addons/web/static/src/scss/form_view.scss

```diff
@@ -611,6 +611,10 @@
             padding-right: 25px;
         }
     }
+    iframe+.o_field_translate {
```

Comment:
I don't understand, there is no `+` in CSS, you can't target something from outside an `iframe`

Reply:
it selects an element right after another (changed it)

Reply:
ah okay I did not know it worked without space so I tought it was a odd `+.` typo